pkg/ebpf/tracer.go

```diff
@@ -166,6 +169,14 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	return pm, nil
 }
 
+// setupExpvars exposes some metrics for tracer
+func (t *Tracer) setupExpvars() {
+	expvar.Publish("tracer.perf_received", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.perfReceived) }))
+	expvar.Publish("tracer.perf_lost", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.perfLost) }))
+	expvar.Publish("tracer.skipped_conns", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.skippedConns) }))
+	expvar.Publish("tracer.expired_tcp_conns", expvar.Func(func() interface{} { return atomic.LoadUint64(&t.expiredTCPConns) }))
+}
```

Comment:
can we expose those https://github.com/DataDog/datadog-agent/blob/master/pkg/ebpf/state.go#L392-L395 too ?

(You can get them using `GetStats` on `t.state`)